_i18n/en.yml

```diff
@@ -147,32 +147,32 @@ team:
   contacts: Chatrooms and Contacts
   chat: "Chat:"
   website: "Website:"
-  core_descr: Stewards of the community, the Core Team manages and maintains infrastructure where centralization cannot be avoided (domains, website, github repositories...). Its members act as mediators as they see fit and aim to weigh consensus in the community, perpetuating the quality interactions that made Monero the successful project it is today. The Core Team does not and has never received any financial compensation.
-  core_start: The Core Team is a closed workgroup; it's not possible to join it unless invited.
-  community_descr: Workgroup used by the community to organize and coordinate. This is the place where every non-coder who wants to contribute to Monero should start.
+  core_descr: As stewards of the project, the Core Team manages and maintains infrastructure where centralization cannot be avoided (domains, website, github repositories...). Its members act as mediators as they see fit and aim to weigh consensus in the community, perpetuating the quality interactions that made Monero the successful project it is today. The Core Team does not and has never received any financial compensation.
```

Comment:
Just pointing out that AFAIK the text was proposed by the core team. The change is not substantial, so i guess it's not a problem.

Reply:
I don't remember that, but regardless "project" is more correct than "community".

Reply:
For the records: https://github.com/monero-project/monero-site/pull/1260#discussion_r510171243. I do agree this version is more correct.